packages/palette/src/elements/FilterSelect/FilterSelect.tsx

```diff
@@ -109,3 +125,50 @@ export const isBelowTheFoldSelected = (selectedItems, resultsSorted) => {
   const isSelected = intersection(selected, results).length > 0
   return isSelected
 }
+
+const SelectAll: React.FC = () => {
+  const {
+    items,
+    filteredItems,
+    onSelectAll,
+    query,
+    selectedItems,
+    setSelectedItems,
+  } = useFilterSelectContext()
+
+  const isClearDisabled = selectedItems.length === 0
+
+  return (
+    <Box my={1}>
+      <Stack gap={2} flexDirection={"row"}>
+        <Clickable
+          data-testid="filterSelect-selectAll"
+          className="selectAll"
+          onClick={() => {
+            setSelectedItems(filteredItems)
+            onSelectAll?.({
+              items,
+              filteredItems,
+              selectedItems: filteredItems,
+              query,
+            })
+          }}
+          textDecoration={"underline"}
```

Comment:
```suggestion
          textDecoration="underline"
```

Reply:
Something about our vsc/prettier setup does this automatically I think, but I agree